### ISSUE TITLE
Look at site for accessibility issues

### ISSUE THREAD

**Issue by tomkeays** [tomkeays/lemoyne#25]
# Accessibility Review

Start with this list -- http://a11yproject.com/checklist.html -- and see what resources Kate Deibel releases as part of the a11y4lib work.

- [x] [Accessibility warning on the "Back to top" button in LibGuides #47](https://github.com/tomkeays/lemoyne/issues/47) - do in tandem with [#63 - Implement LibGuides Recommendations](https://github.com/tomkeays/lemoyne/issues/63)
- [x] [Fix accessibility warning in the Home button in the banner #48](https://github.com/tomkeays/lemoyne/issues/48)
- [x] [Home page template in LibGuides lacks H1 and H2 tags #49](https://github.com/tomkeays/lemoyne/issues/49)
- [ ] [Accessibility: A form control does not have a corresponding label #50](https://github.com/tomkeays/lemoyne/issues/50) - doing this in tandem with [#23 - Assess HTTPS on forms](https://github.com/tomkeays/lemoyne/issues/23) 
- [x] [Increase font size for improved accessibility #54](https://github.com/tomkeays/lemoyne/issues/54)
- [ ] [Fix Accessibility Labeling of SVG Logos #55 ](https://github.com/tomkeays/lemoyne/issues/55)
  - [x] Small Ask Us Icons
  - [x] Large Ask Us Icons
  - [ ] Library Banner Logos
  - [ ] Dolphin Footer Image
- [ ] [Increase Contrast in Banner Navigation #56](https://github.com/tomkeays/lemoyne/issues/56)
- [x] [Fix breadcrumbs across all library domains #68](https://github.com/tomkeays/lemoyne/issues/68)

For another view, see [Accessibility Review Kanban ](https://github.com/tomkeays/lemoyne/projects/2)

**Comment by tomkeays** [comment]
So, yeah, some accessibility issues with links

<img src="https://image.ibb.co/gdzadG/contrast_wcag_aa_small_2.png" alt="contrast_wcag_aa_small_2" border="0">
<img src="https://image.ibb.co/gsZadG/contrast_wcag_aa_small_1.png" alt="contrast_wcag_aa_small_1" border="0">
<img src="https://image.ibb.co/kdxiXb/contrast_wcag_aa_small.png" alt="contrast_wcag_aa_small" border="0">
